controllers/controllers/services/cfservicebinding_controller_test.go

```diff
@@ -230,9 +241,15 @@ var _ = Describe("CFServiceBinding.Reconcile", func() {
 						"Message": Equal(""),
 					})))
 				})
-				It("it patches the existing servicebinding.io ServiceBinding", func() {
+				It("patches the existing servicebinding.io ServiceBinding", func() {
 					Expect(fakeClient.CreateCallCount()).To(Equal(0), "Client.Create call count mismatch")
-					Expect(fakeClient.PatchCallCount()).To(Equal(2), "Client.Patch call count mismatch")
+					Expect(fakeClient.PatchCallCount()).To(Equal(3), "Client.Patch call count mismatch")
+				})
+				It("patches the existing VCAP Services Secret", func() {
+					Expect(fakeBuilder.BuildVCAPServicesEnvValueCallCount()).To(Equal(1))
+					_, appArg := fakeBuilder.BuildVCAPServicesEnvValueArgsForCall(0)
+					Expect(appArg.Name).To(Equal(cfAppName))
+					// Expect patch/update to be called on the VCAP Services Secret from CFApp Status
```

Comment:
I do not see the expectation code that matches that comment. If the expectation code is there, the comment makes no sense

Reply:
I think the comment might have been there to explain why the It statement mentioned patch since the patch call count is verified in the previous It statement. Like mentioned previously, the result of the patch command (Updating the VCAP services secret) is tested in the integration test so the arguments aren't verified in the unit test. I'll remove the comment since it's just creating confusion.